qualtran/bloqs/gf_arithmetic/gf2_inverse.py

```diff
@@ -73,15 +75,22 @@ class GF2Inverse(Bloq):
         result: Output RIGHT register of size $m$ that stores $x^{-1}$ from $GF(2^m)$.
         junk: Output RIGHT register of size $m$ and shape ($m - 2$) that stores
             results from intermediate multiplications.
+
+    References:
+        [Efficient quantum circuits for binary elliptic curve arithmetic:
```

Comment:
Can you keep these links on one line? https://qualtran.readthedocs.io/en/latest/Autodoc.html#references